### DISCUSSION TITLE
关于KOreader阅读器笔记导出的一些建议.

### DISCUSSION THREAD

**Discussion by loganyanger** [koreader/koreader#10325] [Ideas]
可不可以通过插件的形式, 以规定的语法格式, 导出设定特殊排版格式的笔记
例如Calibre 的H2O插件形式, 这样对于OB一站笔记的人群较为友好.
提出建议仅代表个人理解.

**Comment by Frenzie** [comment]
> Is it possible to export specially formatted notes in a specified syntax format through a plugin?
For example, Calibre's H2O plugin, which is more friendly for people who take notes on OB.
Suggestions are made on a personal basis.

The Exporter plugin allows for multiple implementations, see for example https://github.com/koreader/koreader/pull/9165
